src/mandelbrot.rs

```diff
@@ -0,0 +1,68 @@
+// The Computer Language Benchmarks Game
+// http://benchmarksgame.alioth.debian.org/
+//
+// contributed by the Rust Project Developers
+// contributed by TeXitoi
+// contributed by Matt Watson
```

Comment:
If you do not use the original version, you can remove the rust project line.

My contribution must be the last one as, according to the rules of the benchmark game, the last contributor must be the person that submit the benchmark. Or you can keep this order, but then you must propose yourself the benchmark.